googler

```diff
@@ -646,6 +658,9 @@ debug = args.debug
 keywords = args.keywords
 insite = args.insite
 noninteractive = args.noninteractive
+if args.json:
+    output_format = 'json'
+    noninteractive = True
```

Comment:
should we couple json and noninteractive? Users can use both the options together if they ust want to dump the results of a single fetch.